cloudbuild/Dockerfile

```diff
@@ -11,6 +11,9 @@ COPY --chown=ia-tests:ia-tests . /init-actions
 # https://docs.bazel.build/versions/master/install-ubuntu.html
 RUN echo "deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
 RUN curl https://bazel.build/bazel-release.pub.gpg | apt-key add -
-RUN apt-get update && apt-get install -y openjdk-8-jdk python3-setuptools bazel
+RUN apt-get update && apt-get install -y openjdk-8-jdk python3-setuptools bazel libxml2-utils
+
+# Install jq
+RUN apt-get -y install jq	
```

Comment:
Split `apt-get update` and `apt-get install`:
```
RUN apt-get update
RUN apt-get install -y openjdk-8-jdk python3-setuptools bazel libxml2-utils jq
```

Reply:
Done.




> May you attach link with an example how this looks like?

A workflow run looks like this: https://pantheon.corp.google.com/cloud-build/builds/3212df70-c22f-4a0d-98f0-0d7d2dbc1287;step=5?project=cloud-dataproc-ci

TestGrid would look like this: https://screenshot.googleplex.com/jLhd89fyiVj.png